tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -415,6 +471,17 @@ def get_all_test_cases():
             print('\n'.join(get_all_test_cases()))
         return 0
 
+    if args.generate_hrr_tls13_compat_tests:
+        if args.output:
+            with open(args.output, 'w', encoding="utf-8") as f:
+                f.write(SSL_OUTPUT_HEADER.format(
+                    filename=os.path.basename(args.output)))
+                f.write('\n\n'.join(get_hrr_test_cases()))
+                f.write('\n')
+        else:
+            print('\n'.join(get_hrr_test_cases()))
```

Comment:
```suggestion
            print('\n\n'.join(get_hrr_test_cases()))
```
also please change line 471